### PR TITLE
[Matrix] API change update and cleanups

### PR DESCRIPTION
**Pull request by AlwinEsch**
This changes are related to API change with xbmc/xbmc#17813

Need to bring in after them!

### CODE REVIEW

File: inputstream.adaptive/addon.xml.in [hunk 1]
```diff
@@ -13,12 +13,16 @@
...
-	<description lang="es">Cliente InputStream para flujo de datos adaptativos</description>
+    <summary lang="en_GB">InputStream client for adaptive streams</summary>
+    <summary lang="es_ES">Cliente InputStream para flujo de datos adaptativos</summary>
+    <description lang="en_GB">InputStream client for adaptive streams</description>
+    <description lang="es_ES">Cliente InputStream para flujo de datos adaptativos</description>
     <platform>@PLATFORM@</platform>
 	<news>
+v2.5.7 (2020-05-16)
+- Update inputstream API 2.3.1
+- Update Global APi 1.2.0
```
Comment (Sorien) [inline inputstream.adaptive/addon.xml.in]:
```suggestion
- Update Global API 1.2.0
```

Comment (AlwinEsch) [reply inputstream.adaptive/addon.xml.in]: Ah yes, seen. Have them wrongly on most addons.
At home already fixed :smile:. As that after Kodi's request is in, every addon request must be triggered for build, I thought to make with them together.

**Comment by AlwinEsch** [comment]
@peak3d The request on Kodi is in, then this can in too.

**Comment by peak3d** [comment]
thx @AlwinEsch

**Comment by peak3d** [comment]
@AlwinEsch pls. don't bump inputstream.adaptive version in the future.
a.) it is not complete (README.md missing), and b.) I prefer to have an own bump commit without any other changes... Thx!

**Comment by AlwinEsch** [comment]
No problem, I leave it then in future requests away.

**Pull request merged**